web-api/src/business/useCases/courtIssuedOrder/fileCourtIssuedOrderInteractor.test.ts

```diff
@@ -507,4 +473,334 @@ describe('fileCourtIssuedOrderInteractor', () => {
       identifiers: [`case|${caseRecord.docketNumber}`],
     });
   });
+
+  describe('freeText', () => {
+    describe('eventCode "NOT"', () => {
+      it('should add order document to case and set freeText and draftOrderState.freeText to the document title if it eventCode NOT', async () => {
+        await fileCourtIssuedOrderInteractor(
+          applicationContext,
+          {
+            documentMetadata: {
+              docketNumber: caseRecord.docketNumber,
+              documentTitle: 'Order to do anything',
+              documentType: 'Order',
+              draftOrderState: {},
+              eventCode: 'NOT',
+              signedAt: '2019-03-01T21:40:46.415Z',
+              signedByUserId: mockUserId,
+              signedJudgeName: 'Dredd',
+            },
+            primaryDocumentFileId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+          },
+          mockDocketClerkUser,
+        );
+
+        expect(
+          applicationContext.getPersistenceGateway().getCaseByDocketNumber,
+        ).toHaveBeenCalled();
+        expect(
+          applicationContext.getPersistenceGateway().updateCase.mock.calls[0][0]
+            .caseToUpdate.docketEntries.length,
+        ).toEqual(4);
+        expect(
+          applicationContext.getPersistenceGateway().updateCase.mock.calls[0][0]
+            .caseToUpdate.docketEntries[3],
+        ).toMatchObject({
+          draftOrderState: { freeText: 'Order to do anything' },
+          freeText: 'Order to do anything',
+        });
+      });
+    });
+
+    describe('eventCode "O"', () => {
+      it('should add order document to case and set freeText and draftOrderState.freeText correctly for orderType status report', async () => {
+        await fileCourtIssuedOrderInteractor(
+          applicationContext,
+          {
+            documentMetadata: {
+              draftOrderState: {},
+              dueDate: '2024-11-05',
+              eventCode: 'O',
+              orderType: 'statusReport',
```

Comment:
Can we use constants when possible in this file? E.g., here: `STATUS_REPORT_ORDER_OPTIONS.orderTypeOptions.statusReport`

Reply:
Anywhere we have, 'statusReport', 'statusReportStipulatedDecision', 'retained', or 'restoredToGeneralDocket', we should be able to use the `STATUS_REPORT_ORDER_OPTIONS` object.